analysis/museumdraw/analysis1_nov25_vssAbstract.Rmd

```diff
@@ -5,166 +5,223 @@ date: "11/24/2017"
   geom_jitter(alpha=.5) +
-  geom_smooth(method="loess", span=2, alpha=.1)
+  geom_smooth(aes(group = 1), method="loess", span=2, alpha=.1)
 ```
 
 ## Are drawings with more strokes better recognized?
 
-```{r, echo=FALSE}
+```{r}
 ggplot(joint, aes(num_strokes, meanCorrect, color=category)) +
   geom_jitter(alpha=.5) +
-  geom_smooth(method="loess",span=2, alpha=.1)
+  geom_smooth(aes(group = 1), method="loess",span=2, alpha=.1)
 
 ```
 
 ## Are drawings that take longer better recognized?
 
-```{r, echo=FALSE}
+```{r}
 ggplot(joint, aes(draw_duration, meanCorrect, color=category)) +
   geom_jitter(alpha=.5) +
-  geom_smooth(method="loess",span=2, alpha=.1) 
+  geom_smooth(aes(group = 1), method="loess",span=2, alpha=.1) 
 ```
 
 ## Are drawings better recognized with age?
 
-```{r, echo=FALSE}
+```{r}
 ggplot(joint, aes(age, meanCorrect)) +
-  geom_jitter(alpha=.5) +
-  geom_smooth(method="loess",span=2, alpha=.1) 
-
-ggplot(joint, aes(age, meanCorrect, color=category)) +
-  geom_jitter(alpha=.5) +
+  geom_jitter(alpha=.5, width = .1) +
   geom_smooth(method="loess",span=2, alpha=.1) 
 ```
 
 ## How does this trend break down by catergory?
+
 Some items show a stronger trend than others but overall looks good
-```{r, echo=FALSE}
+
+```{r}
 ggplot(joint, aes(age, meanCorrect, color=category)) +
   geom_jitter(alpha=.5) +
   geom_smooth(method="loess",span=2, alpha=.5) + 
   facet_wrap(~category)
 ```
 
+## Which items are easiest and hardest? 
+
+```{r}
+ms <- joint %>%
+  group_by(category) %>%
+  multi_boot_standard(col = "meanCorrect")  %>%
+  ungroup %>%
+  mutate(category = fct_reorder(category, mean))
+
+ggplot(ms, aes(x = category, y = mean)) + 
+  geom_pointrange(aes(ymin = ci_lower, ymax = ci_upper)) + 
+  coord_flip()
+```
+
+## Across ages?
+
+```{r}
+ms <- joint %>%
+  mutate(age_group = cut(age, c(3.9, 6, 8, 10.1), labels = c("4-6","6-8","8-10"))) %>%
+  group_by(category, age_group) %>%
+  multi_boot_standard(col = "meanCorrect")  %>%
+  ungroup %>%
+  mutate(category = fct_reorder(category, mean))
+
+ggplot(ms, aes(x = category, y = mean, col = age_group)) + 
+  geom_pointrange(aes(ymin = ci_lower, ymax = ci_upper)) +
+  coord_flip()
+```
+
 ## How much variance is there per participant (i.e., per child drawer?)
+
 Looks like there are some kids that are very bad or very good at drawing, but there is also a fair number of kids with wide distributions
 
-```{r, echo=FALSE}
-ggplot(joint, aes(1, meanCorrect)) +
-  geom_violin() +
-  facet_wrap(~session_id)
+```{r}
+ms <- joint %>%
+  group_by(session_id) %>%
+  summarise(sd = sd(meanCorrect),
+            mean = mean(meanCorrect))
+
+joint <- left_join(joint, ms) %>%
+  mutate(session_id = fct_reorder(session_id, mean, .desc=TRUE))
+
+ggplot(joint, aes(x = session_id, y = meanCorrect, col = age)) +
+  geom_jitter(width = .1, alpha = .5) + 
+  theme(axis.text.x = element_text(angle = 90, hjust = 1, vjust = .5))
+
 ```
 
 # Inferential statistics: data preprocessing
+
```

Comment:
I try to avoid data transforms - the motto is "model the world correctly, don't transform it into something it's not."